test/features/standup.feature

```diff
@@ -13,57 +8,58 @@ Feature: Standup-module
             | todayMessage     | "Today I'll do this"   |
         Then the response status code should be 201
         And the response should contain:
-            | id               | 123                        |
-            | discordUser      | "eddiehubber"              |
-            | yesterdayMessage | "Yesterday I did this"     |
-            | todayMessage     | "Today I'll do this"       |
-            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | documentId | "TYPE:ID" |
 
     Scenario: search existing standup
         Given authorisation
         And make a POST request to "/standup" with:
             | discordUser      | "eddiehubber"          |
             | yesterdayMessage | "Yesterday I did this" |
             | todayMessage     | "Today I'll do this"   |
+        Then the response should contain:
+            | documentId | "TYPE:ID" |
         Then  make a GET request to "/standup/search?discordUser=eddiehubber"
         Then the response status code should be 200
-        And  the response in property "0" should contain:
-            | id               | 123                        |
-            | discordUser      | "eddiehubber"              |
-            | yesterdayMessage | "Yesterday I did this"     |
-            | todayMessage     | "Today I'll do this"       |
-            | createdOn        | "2021-01-01T00:00:00.000Z" |
+        And the response in item where field "discordUser" is equal to "eddiehubber" should contain:
+            | discordUser      | "TYPE:STRING" |
+            | todayMessage     | "TYPE:STRING" |
+            | yesterdayMessage | "TYPE:STRING" |
```

Comment:
I think for the values that were set in the previous step should be checking the actual value because we know what the values are (from line 16-18)